### PR TITLE
add more info

### PR DESCRIPTION
**Pull request by jpnelson85**
Added info

### CODE REVIEW

**Comment by emlindvall** [comment]
Testing comments here

**Comment by jpnelson85** [comment]
Hi...Test

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
-Adding some text to the README.md 
\ No newline at end of file
+Adding some text to the README.md 
+
+# Git Collaboration Practice
+
```
Comment (emlindvall) [inline README.md:4]: test comment

**Comment by jpnelson85** [comment]
look above

**Pull request merged**